plugins/connection/libvirt_qemu.py

```diff
@@ -11,8 +11,8 @@
 DOCUMENTATION = """
 ---
 author:
-    - Jesse Pretorius <jesse@odyssey4.me>
-connection: community.libvirt.libvirt_qemu
+    - Jesse Pretorius (@odyssey4me)
+name: libvirt_qemu
```

Comment:
@odyssey4me I looked a few other collections and they don't seem to use the full name here, but I'd like to check if you think that this is the right thing to do. Thanks!

Reply:
We originally had this as just the short name, but then it didn't work for someone and it got changed to the long, fully qualified name. I think we should get an answer from the Ansible core/community team here, rather than flip/flop this value.

Reply:
@odyssey4me OK, I'll reach out there. Thanks!

Reply:
This seems to have been addressed here https://github.com/ansible-collections/news-for-maintainers/discussions/11#discussioncomment-2299563 with @felixfontein confirming:

> name: for plugins is the same as module: for modules. It's the 'short name', not a FQCN.